addons/mail/static/src/components/message_seen_indicator/message_seen_indicator.scss

```diff
@@ -2,40 +2,12 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_MessageSeenIndicator {
-    display: flex;
-    position: relative;
-    flex-wrap: nowrap;
```

Comment:
I don't see this rule being moved in template.

I'm not entirely sure why we had it. Maybe it was a technique to avoid the 2nd tick to wrap in other line without the 1st tick. The 2 ticks overlapping with small offset and absolute positioning on 2nd tick can make partial wrapping if we are not careful.

But I fail to get such partial wrapping with my small session of testing. So I'm not entirely sure whether this is relevant or not. At a glance, it's still theoretically possible to wrap partially, which is wrong. Do you agree?

Reply:
I didn't move the `flex-wrap` property because `nowrap` is the default value. So the BS class is not necessary.

Reply:
Oh I see, then it's all good then